docs/current_docs/install.mdx

```diff
@@ -7,3 +7,104 @@ import PartialInstallCli from './partials/_install-cli.mdx';
 # Installation
 
 <PartialInstallCli />
+
+## Update
```

Comment:
We should make some note that you should use the same method that you installed the CLI with in the first place.

Reply:
Done, and then removed the duplicate content.